.gitignore

```diff
@@ -1,23 +1,97 @@
-env
-*.pyc
+# Byte-compiled / optimized / DLL files
```

Comment:
Same comment as the other PR for the gitignore

Reply:
Copied the `.gitignore` from the merged backend's PR

Reply:
Double-checked `.gitignore` file